tests/smc/test_tempered_smc.py

```diff
@@ -65,16 +65,25 @@ def logprior_fn(x):
 
         hmc_kernel = blackjax.hmc.build_kernel()
         hmc_init = blackjax.hmc.init
-        hmc_parameters = extend_params(
-            num_particles,
-            {
-                "step_size": 10e-2,
-                "inverse_mass_matrix": jnp.eye(2),
-                "num_integration_steps": 50,
-            },
+        hmc_parameters_list = [
+            extend_params(
+                num_particles if extend else 1,
```

Comment:
I'd add a comment highlighting that here you are testing that extending with a copy is the same as having a 1 dimension parameter. Otherwise it may be tricky to get why this is even hapening.

Reply:
done